src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -18,6 +18,7 @@
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
```

Comment:
didn't change this either right? not blocking or anything - just confirming so i don't have to compare my changes to this file